src/MCT.js

```diff
@@ -83,7 +83,7 @@ define([
          * @memberof module:openmct.MCT#
          * @name conductor
          */
-        this.conductor = new api.TimeConductor();
+        this.time = new api.Time();
```

Comment:
My intention here is to differentiate between the onscreen "conductor" element which is optionally installed, and the API itself which is always present. Also, "time" is a little more descriptive.